src/MeasureCalculator.java

```diff
@@ -0,0 +1,59 @@
+import java.math.BigDecimal;
+
+public class MeasureCalculator {
+
+    final static int METERS         = 1;
+    final static int CENTIMETERS    = 2;
+    final static int MILIMETERS     = 3;
+    final static int KILOGRAMS      = 1;
+    final static int GRAMS          = 2;
+    final static int MILIGRAMS      = 3;
+
+    private MeasureCalculator(){}
+
+    public static String[] calculateMetric(String inputString, int inputType) {
```

Comment:
Zamiast inputType jako int zrobiłbym to jako enum - ponownie będzie to mieć swoją nazwę i zwiększy czytelność kodu.